polaris/viz/__init__.py

```diff
@@ -0,0 +1,138 @@
+import os
+
+import cmocean  # noqa: F401
+import matplotlib.pyplot as plt
+import numpy as np
+import xarray as xr
+from matplotlib.collections import PatchCollection
+from matplotlib.colors import LogNorm
+from matplotlib.patches import Polygon
+
+
+def plot_horiz_field(config, ds, dsMesh, fieldName, outFileName,
+                     title=None, tIndex=None, zIndex=None,
+                     vmin=None, vmax=None,
+                     cmap=None, cmap_set_under=None, cmap_scale='linear'):
```

Comment:
```suggestion
def plot_horiz_field(config, ds, ds_mesh, field_name, out_filename,
                     title=None, t_index=None, z_index=None,
                     vmin=None, vmax=None,
                     cmap=None, cmap_set_under=None, cmap_scale='linear'):
```
I realize this may not be ready for a full review yet (because of #32) but I would first say let's go with all lowercase variable names throughout.  I've been diligently changing them as I port thing.

Reply:
Will do